### DISCUSSION TITLE
Reg Filter dropdowns SlickGrid

### DISCUSSION THREAD

**Discussion by Gbhuvanesh26** [ghiscoding/Angular-Slickgrid#979] [Q&A]
Hi, 

When in the slick grid the rows are editable and we have our editor dropdown open, and without selecting anything we click on the filter dropdown in the column level, the dropdowns are not closing on click of other filter or even on click of outside until you deliberately click on "OK" in the dropdown. Please let me know if anything can be done regarding this issue.
![image](https://user-images.githubusercontent.com/104485096/182828310-5cf4e1f7-44da-4bb3-b5d1-d116e4b74a74.png)

**Comment by ghiscoding** [comment]
I can only imagine that you're using a very old version of Angular-Slickgrid, this was fixed long time ago. I also cannot help without a reproduction project. I would suggest you upgrade to latest version

**Comment by Gbhuvanesh26** [comment]
Hi @ghiscoding,

Thanks a lot for your quick response, But this is still observed in the latest version of the Angular Slick-grid, 
https://github.com/ghiscoding/Angular-Slickgrid
I've added the link to the GitHub Project we are using
![image](https://user-images.githubusercontent.com/104485096/182887101-14e82ea3-fffe-40c6-ac18-222e9e43cee0.png)

**Reply by ghiscoding** [reply]
Pointing to my very own library is not a valid repro, it works as expected every time that I tried the demo. I won't help if you do not provide a valid repro (a full project that does have the problem). I would suggest you clone the Angular-Slickgrid-Demos repo and try it from there, as far as I can everything is working as expected and no one else mentioned this issue recently
